.github/workflows/build_container.yml

```diff
@@ -0,0 +1,46 @@
+name: build container
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  # build amd64 images
+  jobs-amd64:
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Login to Quay.io Registry
+      run: docker login -u=${{ secrets.DOCKERHUB_USERNAME }} -p=${{ secrets.DOCKERHUB_PASSWORD }} quay.io
+
+    - name: build amd64 image
+      run: ./build_container/docker_build.sh `date +'%Y-%m-%d'`
+
+    - name: create manifest
+      run: |
+        ./build_container/docker_manifest.sh cilium-llvm `date +'%Y-%m-%d'`
+        ./build_container/docker_manifest.sh cilium-iproute2 `date +'%Y-%m-%d'`
+        ./build_container/docker_manifest.sh cilium-bpftool `date +'%Y-%m-%d'`
+
+
+  # build arm64 images
+  jobs-arm64:
+    continue-on-error: true
```

Comment:
Why is this needed?

Reply:
you mean "continue-on-error: true". 

I'm not sure if the community has an Arm64 running environment; 
so add this too prevents a workflow run from failing when arm64 job fails.

Reply:
I see, I guess it makes sense to have this to begin with. Would you mind adding a comment to clarify the intent?